tests/tests.rs

```diff
@@ -896,7 +901,10 @@ fn candid_rpc_should_err_during_restricted_access() {
     setup.clone().as_controller().set_open_rpc_access(false);
     let result = setup
         .eth_get_transaction_receipt(
-            RpcSource::EthMainnet(None),
+            RpcSource::EthMainnet(Some(vec![
+                EthMainnetService::Cloudflare,
+                EthMainnetService::BlockPi,
```

Comment:
BlockPi is still here!   :slightly_smiling_face:

Reply:
Added it back for now (will address the occasional CI issues in another PR if necessary).